api/views.py

```diff
@@ -141,6 +148,7 @@ def get_query_params_description(self):
             'administrator': 'ID of the organization responsible for admin '
                              'work on events.',
             'host': 'ID of the organization hosting the event.',
+            'tag': 'Events\' tag. You can use this parameter multiple times.',
```

Comment:
I would write as `"Events; tags(s). You can use this parameter multiple times."` (double quote to avoid escaping the single quote, then a bit of grammar)